staging/src/k8s.io/apimachinery/pkg/util/httpstream/spdy/roundtripper_test.go

```diff
@@ -22,13 +22,17 @@ import (
 	"encoding/base64"
 	"fmt"
 	"io"
+	"net"
 	"net/http"
 	"net/http/httptest"
 	"net/url"
+	"strconv"
 	"strings"
 	"sync/atomic"
 	"testing"
+	"time"
 
+	"github.com/armon/go-socks5"
```

Comment:
I thought this might be a concern. It's just for test coverage, but it's not ideal. Other packages providing a socks5 server don't look great either. If there's not a suitable test server, I can't include test coverage. What is an acceptable course of action here?

https://github.com/dddpaul/go-socks5-server - 13 commits
https://github.com/ziozzang/socks5-proxy - 9 commits
https://github.com/ring04h/s5.go - 5 commits

Reply:
The lack of activity is not ideal, but there are no non-stdlib transitive dependencies, the repo is small, has an acceptable license, and no churn. I don't see a better alternative, and we wouldn't want to add untested function.